.github/workflows/cloud-workflow.yml

```diff
@@ -0,0 +1,175 @@
+      - name: Lint cloud Go code
+        run: |
+            cd ${MAGMA_ROOT}/orc8r/cloud/docker
+            python3 build.py --lint
+      - name: Generate test coverage
+        run: |
+            cd ${MAGMA_ROOT}/orc8r/cloud/docker
+            python3 build.py --coverage
+      - name: Notify failure to slack
+        if: failure() && github.ref == 'refs/heads/master'
+        uses: rtCamp/action-slack-notify@v2.0.2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_CI }}
+          SLACK_TITLE: "Github action cloud-lint failed"
+          SLACK_USERNAME: "Cloud workflow"
+          SLACK_ICON_EMOJI: ":boom:"
+          SLACK_COLOR: "#FF0000"
+  cloud-test:
+    runs-on: ubuntu-latest
+    env:
+      MAGMA_ROOT: "${{ github.workspace }}"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.6'
+      - name: Run tests cloud Go
+        uses: nick-invision/retry@v2
+        with:
+          command: cd ${MAGMA_ROOT}/orc8r/cloud/docker && python3 build.py --tests --up
+          timeout_minutes: 15
+      - name: Publish Test Results
+        uses: EnricoMi/publish-unit-test-result-action@v1
+        if: always()
+        with:
+          files: ${MAGMA_ROOT}/orc8r/cloud/test-results
+      - name: Notify failure to slack
+        if: failure() && github.ref == 'refs/heads/master'
+        uses: rtCamp/action-slack-notify@v2.0.2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_CI }}
+          SLACK_TITLE: "Github action cloud-test failed"
+          SLACK_USERNAME: "Cloud workflow"
+          SLACK_ICON_EMOJI: ":boom:"
+          SLACK_COLOR: "#FF0000"
+  orc8r-build:
+    runs-on: ubuntu-latest
+    needs: [insync-checkin, deploy-sync-checkin, cloud-lint, cloud-test]
+    if: github.ref == 'refs/heads/master'
+    env:
+      MAGMA_ROOT: "${{ github.workspace }}"
+      DOCKER_REGISTRY: "${{ secrets.DOCKER_REGISTRY }}"
+      DOCKER_USERNAME: "${{ secrets.DOCKER_USERNAME }}"
+      DOCKER_PASSWORD: "${{ secrets.DOCKER_PASSWORD }}"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run apt-get update
+        run: sudo apt-get update
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.6'
+      - name: Run build.py
+        run: |
+            cd ${MAGMA_ROOT}/orc8r/cloud/docker
+            python3 build.py --all --nocache --parallel
+      - name: Tag and push to Docker Registry
+        run: |
+            ./ci-scripts/tag-push-docker.sh -i 'nginx|controller' -t '${GITHUB_SHA:0:8}' -tl true -p orc8r
+      - name: Tag and push to Jfrog Registry
+        env:
+          DOCKER_REGISTRY: "${{ secrets.JFROG_DOCKER_ORC8R_REGISTRY }}"
+          DOCKER_USERNAME: "${{ secrets.JFROG_USERNAME }}"
+          DOCKER_PASSWORD: "${{ secrets.JFROG_PASSWORD }}"
+        run: |
+            ./ci-scripts/tag-push-docker.sh -i 'nginx|controller' -t '${GITHUB_SHA:0:8}' -tl true -p orc8r
```

Comment:
Can you add long options

Reply:
+1. As a general rule of thumb, use long flags everywhere as it increases readability https://magma.github.io/magma/docs/next/contributing/contribute_conventions#all-documentation